components/media-player/MediaPlayerOpen.vue

```diff
@@ -24,22 +26,68 @@ const {
   rewind,
   fastForward,
   repeatStatus,
+  volume,
 } = useNuxtApp().$mediaPlayer;
 
-const onPointerUpProgressBar = (event: PointerEvent) => {
-  const rect = (event.currentTarget as Element)?.getBoundingClientRect();
-  currentPosition.value =
-    ((event.clientX - rect.left) / rect.width) * currentTrackDuration.value;
+const [state, send] = useMachine(
+  slider.machine({
+    id: "volume",
```

Comment:
@andreasgangso I refactored the volume slider. Do you by chance know how to make a 2nd slider work?
I haven't really tried it much but my first try caused some problems.

Reply:
done